mailscanner/functions.php

```diff
@@ -308,10 +299,18 @@ function html_start($title, $refresh = 0, $cacheable = true, $report = false)
         }
     }
 
-    if (!isset($_SESSION['last_update']) || $_SESSION['last_update'] + SESSION_TIMEOUT < time()) {
+    // Check for a privilege change
+    if (checkPrivilegeChange($_SESSION['myusername']) === true) {
+        header('Location: logout.php?error=timeout');
+    }
```

Comment:
here a `die()` is needed, otherwise the code following `header()` will be execute